articles/azure-functions/functions-bindings-timer.md

```diff
@@ -287,7 +287,9 @@ Expressed as a string, the `TimeSpan` format is `hh:mm:ss` when `hh` is less tha
 |---------|---------|
 |"01:00:00" | every hour        |
 |"00:01:00"|every minute         |
-|"24:00:00" | everyday        |
+|"24:00:00" | every 24 days        |
+|"1:00:00:00:00" | everyday |
```

Comment:
Is there a source that you can reference that documents the format `1:00:00:00:00`? This doesn't seem to follow the standard `hh:mm:ss` or `dd:hh:mm` format.

Reply:
@yoichinak - The correct notation should be as in my change request, which was marked as duplicate: "1.00:00:00"

```suggestion
|"1.00:00:00" | every day        |
```

You can find the official documentation, [here](https://docs.microsoft.com/en-us/dotnet/api/system.timespan?view=netframework-4.8). I'm including the relevant, official blurb for ease and readability's sake:

_A TimeSpan value can be represented as **[-]d.hh:mm:ss.ff**, where the optional minus sign indicates a negative time interval, **the d component is days**, hh is hours as measured on a 24-hour clock, mm is minutes, ss is seconds, and ff is fractions of a second. That is, a time interval consists of a positive or negative number of days without a time of day, or a number of days with a time of day, or only a time of day._ 

Note that the separator between days and hours is a decimal and you can verify that this should be the expected value in the .NET source code. 

See:
https://referencesource.microsoft.com/#mscorlib/system/globalization/timespanparse.cs,352
https://referencesource.microsoft.com/#mscorlib/system/globalization/timespanformat.cs,337

If you have any questions or need anything further, feel free to reach out!